frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts

```diff
@@ -192,6 +194,17 @@ export const playerSettingsLogic = kea<playerSettingsLogicType>([
             if (values.speed !== playerSpeed) {
                 actions.setSpeed(playerSpeed)
             }
+            const showLLMMetadata = 'showLLMMetadata' in searchParams && (searchParams.showLLMMetadata ?? false)
+            if (values.showLLMMetadata !== showLLMMetadata) {
+                actions.setShowLLMMetadata(showLLMMetadata)
+            }
+        },
+        // Putting `*` last to match it only if more specific routes don't match, as the matching seems to be exclusive
```

Comment:
@pauldambra The logic seems broken now in master, as matching `*` stops any follow-up matches (so anything defined for `export/shared` felts like not working (at least, locally) 🤔 If it was correct - moving `*` lower should not break anything anyway?